docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -104,9 +126,15 @@ The minimum supported Node.js version has been changed from 12.0.0 to 14.0.0, si
   | `GridAggregationColumnMenuItem`                                     | `GridColumnMenuAggregationItem` |
   | `GridRowGroupingColumnMenuItems`, `GridRowGroupableColumnMenuItems` | `GridColumnMenuGroupingItem`    |
 
-- `GridFilterItemProps` has been renamed to `GridColumnMenuItemProps`.
+- The `GridFilterItemProps` has been renamed to `GridColumnMenuItemProps`.
 - Props `column` and `currentColumn` passed to `GridColumnMenu` and column menu items have been renamed to `colDef`
 
+These changes are not part of `preset-safe` codemod and you need to handle them manually:
```

Comment:
IMHO: This calls for a separate section to increase visibility of this fact.
But I'd suggest approaching further migration docs refactor with a separate PR.
You could consider following the text style similar to MUI Core or pickers, where migration doc (sections) tell the user what will need to be done to migrate, instead of what has been done.

Reply:
[This case](https://mui.com/material-ui/migration/v5-component-changes/#remove-withmobiledialog-component) in core seemed similar where part of the change was included in `preset-safe`, handled our case the same way by removing ✅ and adding a warning so that users are aware that some extra work may be needed.

Reply:
Gotcha. That's also a solution, especially if it worked for MUI Core. 👍 
Although, I'd consider splitting it into a separate section or two different sub-sections (like with MUI Core: Component -> specific change).
Or maybe using a different icon, like you mentioned, something along the lines of a warning, or maybe a checkmark and a warning. 🤷 
But in any case, those are just thinking out loud nitpicks. 😉

Reply:
Sure thanks for sharing the ideas. They are good and we can consider them with upcoming PRs 👍